BugsnagNetworkRequestPlugin.podspec.json

```diff
@@ -12,7 +12,7 @@
     "tag": "v6.11.0"
   },
   "dependencies": {
-    "Bugsnag": "~> 6.10"
+    "Bugsnag": "6.12.1"
```

Comment:
This reverts a [fix](https://github.com/bugsnag/bugsnag-cocoa/commit/3d4c9a05105b94bf64c1817091d31bea5691139e) that was made to address build errors in the example apps.